app/views/users/edit.js.erb

```diff
@@ -0,0 +1,8 @@
+$('.alert-danger').remove()
+$('.hbx_profiles').append("<%=escape_javascript(render partial: 'shared/modals/edit_email_form', locals: { user: @user }) %>")
+
+$('#setUserEmailModal').modal({
+    backdrop: 'static',
```

Comment:
I don't think these are necessary - I used those properties to make the security question modal unescapable. This one we allow to close though, so it's not needed.

Not an important change, however, since there is still a close button